Jenkinsfile

```diff
@@ -75,14 +77,14 @@ pipeline {
             }
             }
         }
-        stage("pytest3"){
-        agent{  label utils.rocmnode("tunatest") }
-        steps{
-            script{
-            utils.pytestSuite3()
-            }
+        stage("pytest3 and Tests Coverage"){
+            agent { label utils.rocmnode("tunatest") }
+            steps {
+                script {
+                    utils.pytestSuit3AndCoverage(branch, branch_master)
```

Comment:
I might be missing something, but can't we put the generation of coverage report into a separate stage?

Reply:
thought about doing that...opted to keep it with pytest3suite

Reply:
I'm not sure how putting them in a single stage helps? Since they are two separate tests, wouldn't it be better to carry them out in separate stages? If the coverage test fails, you can just rerun the coverage stage, instead of having to rerun both PytestSuite3 and Coverage.

(Also, there seems to be a slight misspelling in pytestSuit3)

Reply:
Spelling Fixed. The rest is feasible for a separate PR for the current timing.